### ISSUE TITLE
Improve rule S2068 to detect password in db connections

### ISSUE THREAD

**Issue by saberduck** [SonarSource/sonar-go-archived#381]
Rule [S2068](https://rules.sonarsource.com/go/RSPEC-2068) should detect when credentials are hardcoded in the db connection string

```go
package main

import (
    "database/sql"
)

// connect Database with hardcoded values
func connectDatabase() (*sql.DB, error) {
    var err error

    db, err := sql.Open("postgres", "user=notactualusername password=notactual dbname=aggregatordb sslmode=disable host=aggregatordb-primary") // Noncompliant

    if err != nil {
        panic("Database connection failure")
    }

    return db, nil
} 

```
